File: src/main/java/com/loltft/rudefriend/RudeFriendApiApplication.java
package com.loltft.rudefriend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class RudeFriendApiApplication {

  public static void main(String[] args) {
    SpringApplication.run(RudeFriendApiApplication.class, args);
  }
}

File: src/main/java/com/loltft/rudefriend/RudeFriendApiApplication.kt
package com.loltft.rudefriend

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
@ConfigurationPropertiesScan("com.loltft.rudefriend")
class RudeFriendApiApplication

fun main(args: Array<String>) {
    runApplication<RudeFriendApiApplication>(*args)
}